ui/src/pages/QuickSetup/AssignRolesPage.tsx

```diff
@@ -366,45 +313,36 @@ const makeInstruction = (inst: InstFieldsWithType): OnboardingInstruction => {
 const getFields = (inst: InstFieldsWithType) => {
   const provider = FieldType.PARTIES;
   const custodian = FieldType.PARTIES;
-  const tradingAccount = FieldType.TEXT;
   const clearingAccount = FieldType.TEXT;
```

Comment:
can `clearingAccount` be removed too?

Reply:
`ClearingAccount` is the only remaining account at the `Service` level. Luciano is working on clearing which was why I tried to make the least changes to clearing as possible